broker/consumer/vm_startup.sh

```diff
@@ -3,9 +3,7 @@
 
 brokerdir="/home/broker"
 workingdir="${brokerdir}/consumer"
-# the keytab file required for authorization must already exist in the workingdir.
-# make it immutable, then delete everything else so we can start fresh.
-chattr +i "${workingdir}/pitt-reader.user.keytab"
+# delete everything so we can start fresh
 rm -rf "${brokerdir}"
```

Comment:
I got the following message when I entered this command (ssh'd in):
rm: cannot remove '/home/broker/consumer/pitt-reader.user.keytab': Operation not permitted

Reply:
Oh, this must be because we already made that file immutable. Try making it mutable again and then just deleting it. Do this manually, so:  ssh in, do `chattr -i "${workingdir}/pitt-reader.user.keytab"`, then `rm "${workingdir}/pitt-reader.user.keytab"`.

Reply:
We don't need the keytab file at all anymore, so you really do want to delete it altogether. Nothing should need to be changed in the scripts. This is a secret file that should not ever be committed to the repo, but I have a local copy (and probably you do too) if we do need in the future for some reason.

Reply:
got it, thanks!